### DISCUSSION TITLE
Failed to send shoutrrr notification - Discord

### DISCUSSION THREAD

**Discussion by masterwishx** [containrrr/watchtower#1776] [Q&A] [answered]
Using Oracle free cloud in portainer stack:

discord://token@id

![image](https://github.com/containrrr/watchtower/assets/28630321/6cd2f742-81de-4af0-a233-2660edbcd273)

![image](https://github.com/containrrr/watchtower/assets/28630321/76795458-ee17-4669-835f-fad0a92f0c59)

**Comment by masterwishx** [comment, marked as answer]
Fixed, setting was not right :
discord://id@token

**Comment by Hussienfahmy** [comment]
What is the token and what is the ID ?
